packages/components/src/form/src/Form.css

```diff
@@ -33,6 +25,8 @@
     margin-bottom: var(--o-ui-sp-3);
 }
 
-.o-ui-fieldset .o-ui-form-field:last-child {
-    margin-bottom: 0;
+.o-ui-fieldset-fields {
+    display: grid;
```

Comment:
A grid display without any specific templating doesn't seem right.

Reply:
This makes gap works ;) Flex could be used here too, Grid has the advantage of not messing with the layout direction.

Reply:
Yeah I know, I guess I would have choose Flex since there isn’t any expected templating here.

That being said, should a fieldset really have any layout? I think it’s the content of the fieldset which should be responsable for the layout.

Reply:
It should be a Flex or Grid layout inside the Fieldset which provide the spacing between the fields.

Fieldsets is only there to provide semantic meaning not layout.

Reply:
This means the end user will need to add a wrapper, setting the field set as a flex layout with a gap takes this off the consumer hands. Free responsive layout 
![image](https://user-images.githubusercontent.com/361632/154991868-b475b237-9fe4-41b1-a418-c3d3de956e0e.png)

Reply:
I am not sure I understand what you mean but the code sample for Fieldset already includes a Grid so what else does the user will need to add?

![image](https://user-images.githubusercontent.com/794579/154993639-79c24e54-90df-45f4-90dd-983d6db7ec46.png)

Reply:
The right amount of gap. Or fieldset could inherit gap?

Reply:
What do you mean by the right amound of gap?

My current guess is that you are trying a problem I am not aware exist.